api/src/test/java/marquez/service/LineageServiceTest.java

```diff
@@ -158,9 +158,6 @@ public void testLineage() {
     runAssert
         .extracting(Run::getInputVersions, InstanceOfAssertFactories.list(DatasetVersionId.class))
         .hasSize(0);
-    runAssert
-        .extracting(Run::getOutputVersions, InstanceOfAssertFactories.list(DatasetVersionId.class))
-        .hasSize(1);
```

Comment:
We are asserting on the `Lineage` object returned from `lineageService.lineage` call, and it is the response payload of GET lineage API. Does this mean that API response is being changed?

Reply:
Yeah, I don't like all the information being returned here, but breaking API compatibility is not good. If we want a lighter-weight version of the lineage API, I think it's better to either include an optional parameter to exclude the superfluous data or to create a new API and deprecate the old one

Reply:
I have updated the PR and added a `withRunFacets` flag to get runs with all the superfluous data and this flag always true in Marquez apis, so no affect on the API compatibility.